core/drivers/crypto/caam/hal/ls/hal_clk.c

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2019 NXP
+ *
+ * Brief   CAAM Clock functions.
+ */
+
+/* Local includes */
+#include <caam_hal_clk.h>
+
+/*
+ * Enable/disable the CAAM clocks
+ *
+ * @enable  Enable the clock if true
+ */
+void hal_clk_enable(bool enable __unused)
+{
+}
+
```

Comment:
All the `hal/ls/*` files can probably be dropped since this PR only adds support for imx. They can be readded later with `ls` support.

Reply:
It can not be removed as it's a platform already supported and we have aligned with the team to ensure that we are upstreaming support to enable them too.